charts/gsp-cluster/templates/02-gsp-system/cloudhsm-service.yaml

```diff
@@ -0,0 +1,88 @@
+{{ if .Values.global.cluster.cloudHSMExpose }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-haproxy
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: vmc
```

Comment:
Think this should probably not be `vmc` .... maybe `hsmproxy` (see previous comment)

Reply:
Thanks - fixed.